Source/Core/DolphinQt2/GameList/GameTracker.cpp

```diff
@@ -21,6 +23,11 @@ GameTracker::GameTracker(QObject* parent) : QFileSystemWatcher(parent)
   connect(this, &QFileSystemWatcher::directoryChanged, this, &GameTracker::UpdateDirectory);
   connect(this, &QFileSystemWatcher::fileChanged, this, &GameTracker::UpdateFile);
 
+  cache.Load();
```

Comment:
There should be a matching call to `cache.Save()` somewhere, probably after every call to `cache.Update()` (to match DolphinWX's behavior).